app/src/pages/trace/AnnotationActionMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { ActionMenu, Flex, Icon, Icons, Item, Text } from "@arizeai/components";
+
+type AnnotationActionMenuProps = {
+  onDelete: () => void;
+  isDisabled?: boolean;
+};
+
+enum AnnotationAction {
+  DELETE = "deleteAnnotation",
+}
+
+/**
+ * A generic action menu for annotations that can be extended
+ */
+export function AnnotationActionMenu(props: AnnotationActionMenuProps) {
+  const { onDelete, isDisabled = false } = props;
+  return (
+    <div
+      // TODO: add this logic to the ActionMenu component
```

Comment:
is this a ui-components follow up?

Reply:
Yup: https://github.com/Arize-ai/ui-components/issues/226